src/app/merge-diff/merge-diff-container/merge-diff-container.component.ts

```diff
@@ -32,9 +36,14 @@ import { SharedService, StateHandlerService } from '@mdm/services';
 })
 export class MergeDiffContainerComponent implements OnInit {
 
+  loaded = false;
+  domainType: string;
```

Comment:
Maybe use `CatalogueItemDomainType` instead (unless there is a specific reason it's a generic string).